src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/ManagedIndexRunner.kt

```diff
@@ -217,15 +242,14 @@ object ManagedIndexRunner : ScheduledJobRunner,
         val roles = managedIndexConfig.getRoles()
         logger.debug("Running ISM job: ${managedIndexConfig.name} with roles: $roles Thread: ${Thread.currentThread().name}")
 
-        // Get current IndexMetaData and ManagedIndexMetaData
-        val indexMetaData = getIndexMetaData(managedIndexConfig.index)
-        if (indexMetaData == null) {
-            return
-        }
-        val managedIndexMetaData = indexMetaData.getManagedIndexMetaData()
+        val indexMetaData = getIndexMetadata(managedIndexConfig.index) ?: return
```

Comment:
Instead of just return can we actually log out a warning that it failed to find index metadata for this index?

Reply:
add a warning log